library/OrderCloud.Catalyst/Auth/UserAuth/OrderCloudUserAuth.cs

```diff
@@ -23,7 +17,7 @@ namespace OrderCloud.Catalyst
 	/// </summary>
 	public class OrderCloudUserAuthAttribute : AuthorizeAttribute
 	{
-		public string[] OrderCloudRoles => Roles.Split(",");
+		public List<string> OrderCloudRoles => Roles?.Split(",").ToList() ?? new List<string> { };
```

Comment:
I think you need an `?.` on ToList() in case the previous evaluates to null

Reply:
Good call, thanks!